app/Http/Controllers/V2/AuditStatus/GetAuditStatusController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Http\Controllers\V2\AuditStatus;
+
+use App\Http\Controllers\Controller;
+use App\Http\Resources\V2\AuditStatusResource;
+use App\Models\V2\AuditStatus\AuditStatus;
+use App\Models\V2\Projects\Project;
+use App\Models\V2\Sites\Site;
+use App\Models\V2\Sites\SitePolygon;
+use Illuminate\Http\Request;
+
+class GetAuditStatusController extends Controller
+{
+    public function __invoke(Request $request, string $id = null)
+    {
+        $auditStatus = AuditStatus::where('entity', $request->input('entity'))
+            ->where('entity_uuid', $request->input('uuid'))
```

Comment:
It looks like this is doing a polymorphic association, but it isn't using any of the tools we have in place to do these gracefully. 

This should be set up like so:
 * In `AuditAttachment`, the columns should be named something like `auditable_type` and `auditable_id`
 * A `morphTo` association should be created (see `Seeding.php` for a simple example)
 * The API endpoint should be structured with the entity type and ID in the URL (`/api/v2/audit-status/sites/{uuid}`)
 * With that done, the middleware for safely retrieving polymorphic associates can be used.
     * Create an interface (likely called `Auditable` - can be empty if there aren't any useful methods to include; I haven't looked deeply enough yet to tell. At the least, an association with auditStatuses would likely be helpful, like a `morphMany` accessor - see `SiteReport->seedings()`) and implement it on any model that AuditStatus can be attached to
     * In `ModelBindingInterfaceMiddleware`, make sure all of the classes that are Auditable are listed in `CONCRETE_MODELS` (from what I see here, likely just `SitePolygon` needs to be added)
     * The API definition for this route in `api_v2` then becomes something like
       ```
       Route::prefix('audit-status')->group(function () {
             ModelInterfaceBindingMiddleware::with(Auditable::class, function () {
                Route::get('/{auditable}', GetAuditStatusController::class);
            });
       ```
     * This method definition changes to
       ```
       public function __invoke(Request $request, Auditable $auditable) {
       ```

This method becomes smaller and more efficient (in combination with comments below; especially dropping the need for the N+1 `getEntity` query would be especially good)
```
$auditStatuses = auditable->auditStatuses()
  ->orderBy('updated_at', 'desc')
  ->orderBy('created_at', 'desc')
  ->with('auditAttachments')
  ->get();
return AuditStatusResource::collection($auditStatuses); 
```
return AuditStatusResource::collection(auditable->